components/sections/Projects/index.jsx

```diff
@@ -1,32 +1,44 @@
+/* eslint-disable @next/next/no-img-element */
 import projectsData from '/data/projects'
 import Title from '../../ui/Title'
 import Wrapper from '../Wrapper'
 import ProjectItem from './ProjectItem'
 import { useState } from 'react'
 
 const Projects = () => {
-    const [active, setActive] = useState(0);
+    const [active, setActive] = useState();
 
   return (
     <Wrapper>
-        <div className="mt-34">
+        <div className="my-34 xl:pb-137">
             <Title name="Projects" />
-            <div className="xl:flex justify-between">
-                <div className="flex flex-col relative mt-2.75">
+            <div className="xl:flex items-start justify-between mt-2.75">
+                <div className="xl:h-48">
+                    <div className="xl:absolute">
+                        <div className="flex flex-col relative xl:gap-6.5">
+                            {
+                                projectsData.map((project, index) =>(
+                                    <ProjectItem {...project} key={index} index={index} setActive={setActive} active={active} />
+                                ))
+                            }
+                        </div>
+                    </div>
+                </div>
+                <div className="hidden xl:block sticky w-full top-10 bg-red-600">
                     {
                         projectsData.map((project, index) =>(
-                            <ProjectItem {...project} key={index} index={index} setActive={setActive} active={active} />
+                            <div
+                                className={`overflow-hidden absolute right-0 max-w-202.75 transition-all duration-700 drop-shadow-pr lg:-z-10 ${active === index ? 'opacity-100': 'opacity-0'}`}
```

Comment:
Instead of using single quotes, use double quotes to maintain the same style format